### PR TITLE
add Watson discovery notebook

### PR DESCRIPTION
**Pull request by MichaelClifford**
## Related Issues and Dependencies

#40 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Adding the incomplete demo notebook from the Watson NLU workshop that we will modify to recreate out keyword analysis work.

### CODE REVIEW

**Comment by oindrillac** [comment]
@MichaelClifford thanks. Just a small nit - can we add a url to the source of the notebook/workshop just for code referencing purposes? :slightly_smiling_face:

**Comment by MichaelClifford** [comment]
@oindrillac  added the following markdown to the top of the notebook:'

image

**Review by oindrillac — APPROVED** [review]
:ship:

File: notebooks/stage/watson_keywords.ipynb [hunk 1]
```diff
@@ -0,0 +1,1355 @@
+{
```
Comment (tumido) [inline notebooks/stage/watson_keywords.ipynb:1]: <p>Can we load these values from environment if they are not defined or it's executed from automation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (tumido) [inline notebooks/stage/watson_keywords.ipynb:1]: <p>I think the prints here deserve a function. Something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">from pprint import pprint

def describe_result(heading, obj):
    print(f'"{heading}"\n')
    pprint(obj, indent=3)
</pre><p> </p><p>Or you can even consider printing it as a table from a pandas data frame... The data is fortunately consistent, so pandas should be able to auto-format it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (tumido) [inline notebooks/stage/watson_keywords.ipynb:1]: <p>This reminds me that we should update our license...</p><p> </p><p>https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/blob/master/LICENSE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Comment by tumido** [comment]
if my comments are missing the point and you're just simply trying to copy a notebook over in here, feel free to dismiss my review :slightly_smiling_face:

**Comment by MichaelClifford** [comment]
Hey @tumido,  all good points :smile: , and we should incorporate them into upcoming PR's on this notebook. But as you pointed out, this specific PR was just to include an existing notebook that @oindrillac and I are going to modify to meet our needs for testing out some IBM Watson NLU/Discovery features. 

The ultimate goal for this notebook though should be to add another "analyses" notebook to our workflow  (see #40)

**Pull request merged**